lib/lago/__init__.py

```diff
@@ -464,46 +465,67 @@ def _create_disk(
 
             disk_filename = '%s_%s.%s' % (name, spec['name'], spec['format'])
             disk_path = self.paths.images(disk_filename)
+
             if spec['type'] == 'template':
-                if template_store is None or template_repo is None:
-                    raise RuntimeError('No templates directory provided')
+                template_type = spec.get('template_type', 'repository')
+                if template_type == 'repository':
```

Comment:
better use lago as template type here, repository is too broad (virt-builder uses also a repo, vagrant too, glance is an image repo too)